test/providers/provider_level_zero.cpp

```diff
@@ -423,6 +435,36 @@ TEST_P(umfLevelZeroProviderTest, levelZeroProviderNullParams) {
     EXPECT_EQ(res, UMF_RESULT_ERROR_INVALID_ARGUMENT);
 }
 
+TEST_P(umfLevelZeroProviderTest, setDeviceOrdinalValid) {
```

Comment:
would be also nice to have a test for invalid case

Reply:
Yeah, but there's a problem - it seems L0 does do validate the ordinals right now so even if you pass invalid one, you might succeed.

Reply:
hmm I see, so at least you could add negative test for null params (see testcase above)

Reply:
Done.